rust/operator-binary/src/controller.rs

```diff
@@ -1782,7 +1793,7 @@ mod tests {
           name: trino
         spec:
           image:
-            productVersion: "455"
+            productVersion: "469"
```

Comment:
```suggestion
            productVersion: "470"
```

Reply:
Done in 46b26f1

Reply:
Done in dbf9409e08c5b82aff7d4724af1fa5b014e96fbf